stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -190,6 +190,21 @@
       description = "When set, it binds web services to listen address only")
   boolean bindToListenAddressOnly = false;
 
+  @Order(value = 16)
+  @Option(
+      name = {"--jmx-port"},
+      description = "The port on which JMX should start")
+  int jmxPort = 7199;
+
+  @Order(value = 17)
+  @Option(name = {"--dynamic_snitch", "Set if the dynamic snitch should wrap the actual snitch."})
+  boolean dynamicSnitch = true;
+
+  @Order(value = 18)
+  @Option(
+      name = {"--disable_mbean_registration", "Whether the mbean registration should be disabled"})
+  boolean disableMBeanRegistration = false;
```

Comment:
The `starctl` script needs updated to support these.

Reply:
done in: https://github.com/stargate/stargate/pull/93/commits/d00e650a0f6c2768b3e43ef588fbc354c29f4496 but one condition: `dynamic_snitch` has default `true`. So passing this `--dynamic_snitch` will not change the default. Do we have a clear way to pass the negation?

Reply:
In the script if the value is not provided then you pass in `true` to the starter?